static/js/settings_config.ts

```diff
@@ -492,6 +492,7 @@ export const display_settings_labels = {
             defaultMessage: "Convert emoticons before sending (<code>:)</code> becomes 😃)",
         }),
     ),
+    display_organization_name: $t({defaultMessage: "Display organization name"}),
```

Comment:
This setting name needs to be more specific.

Reply:
How about "Display organization name in left-sidebar"?